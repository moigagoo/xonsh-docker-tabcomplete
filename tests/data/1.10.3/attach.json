{
        "arguments": [
                "--detach-keys",
                "--help",
                "--no-stdin",
                "--sig-proxy=true"
        ],
        "commands": []
}
