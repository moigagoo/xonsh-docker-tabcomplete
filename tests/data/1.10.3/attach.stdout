
Usage:	docker attach [OPTIONS] CONTAINER

Attach to a running container

  --detach-keys       Override the key sequence for detaching a container
  --help              Print usage
  --no-stdin          Do not attach STDIN
  --sig-proxy=true    Proxy all received signals to the process
